generator/graphql-kotlin-federation/src/test/kotlin/com/expediagroup/graphql/generator/federation/validation/integration/FederatedRequiresDirectiveIT.kt

```diff
@@ -70,7 +71,7 @@ class FederatedRequiresDirectiveIT {
         assertEquals(expected, exception.message)
     }
 
-    @Test
+    @Disabled("this is a valid use case according to Federation 2 spec")
```

Comment:
instead of disabling this test, this code (and referenced sample project) should be deleted